tests/client/amo/components/TestMoreInfo.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  findRenderedComponentWithType,
+  renderIntoDocument,
+} from 'react-addons-test-utils';
+import { findDOMNode } from 'react-dom';
+import { Provider } from 'react-redux';
+
+import createStore from 'amo/store';
+import { MoreInfoBase } from 'amo/components/MoreInfo';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+describe('<MoreInfo />', () => {
+  const initialState = { api: { clientApp: 'android', lang: 'pt' } };
+  const addon = {
+    current_version: { id: 20 },
+    homepage: 'http://hamsterdance.com/',
+    has_privacy_policy: true,
+    slug: 'my-addon',
+  };
+
+  function render(props) {
+    return findRenderedComponentWithType(renderIntoDocument(
+      <Provider store={createStore(initialState)}>
+        <MoreInfoBase i18n={getFakeI18nInst()} addon={addon} {...props} />
+      </Provider>
+    ), MoreInfoBase);
+  }
+
+  it('renders loading when loading more info', () => {
+    const rootNode = findDOMNode(render({ versionDetails: { loading: true } }));
+
+    assert.equal(rootNode.querySelector('.MoreInfo-contents').textContent,
+      'Loading...');
+  });
+
+  it('does not render a homepage if none exists', () => {
```

Comment:
I find that tests like this become brittle over time. For example, some new data could be added to `addon` that is unrelated to this test. This might break the test. You could keep it robust by taking an approach like this:
````javascript
import deepcopy from 'deepcopy';
const partialAdon = deepcopy(addon);
delete partialAddon.homepage;
// Run the test with partialAddon
````
See: https://www.npmjs.com/package/deepcopy